src/main/java/org/folio/rest/helper/FundsHelper.java

```diff
@@ -241,16 +241,16 @@ private Future<Void> updateFundGroups(CompositeFund compositeFund, RequestContex
           return getGroupFundFiscalYearsThatFundBelongs(fund.getId(), currentFiscalYearId)
             .compose(groupFundFiscalYearCollection -> {
               List<String> groupIdsFromStorage = StreamEx.of(groupFundFiscalYearCollection).map(GroupFundFiscalYear::getGroupId).collect(Collectors.toList());
-              log.info("updateFundGroups:: Creating group fund fiscal years for new groups");
+              log.info("updateFundGroups:: Creating group fund fiscal years for new groups for compositeFund '{}'", compositeFund.getFund().getId());
```

Comment:
we already have variable fund, lets use it

Reply:
Thanks, used